src/backend/api/Fusion.Resources.Api/Controllers/Person/PersonController.cs

```diff
@@ -28,11 +30,13 @@ public class PersonController : ResourceControllerBase
     {
         private readonly IHttpClientFactory httpClientFactory;
         private readonly IFusionProfileResolver profileResolver;
+        private readonly IMediator mediator;
 
-        public PersonController(IHttpClientFactory httpClientFactory, IFusionProfileResolver profileResolver)
+        public PersonController(IHttpClientFactory httpClientFactory, IFusionProfileResolver profileResolver, IMediator mediator)
```

Comment:
This is handled in the base controller.. You can use it by just using the protected Dispatch method..